### PR TITLE
validation

### PR DESCRIPTION
**Pull request by yCobanoglu**

### CODE REVIEW

File: ebl/fragmentarium/application/fragment_finder.py [hunk 1]
```diff
@@ -41,8 +41,8 @@ def search(self, number: str) -> List[FragmentInfo]:
         )
 
     def inject_documents_in_fragment_infos(
```
Comment (jlaasonen) [inline ebl/fragmentarium/application/fragment_finder.py:43]: The name is misleading. The function does the search. Injecting is just an implementation detail.

File: ebl/fragmentarium/web/fragment_search.py [hunk 2]
```diff
@@ -25,9 +22,7 @@ def __init__(
         self._dispatch = create_dispatcher(
             {
                 frozenset(["id", "pages"]): lambda value:
-                finder.inject_documents_in_fragment_infos(
-                    *self._validate_pages(**value)
```
Comment (jlaasonen) [inline ebl/fragmentarium/web/fragment_search.py:29]: The validation was removed. What happens if the input does not represent a number?

Comment (yCobanoglu) [reply ebl/fragmentarium/web/fragment_search.py:29]: ok will leave the validation i though validating on the frontend is enough

Comment (jlaasonen) [reply ebl/fragmentarium/web/fragment_search.py:29]: Never trust data from the web.

**Pull request merged**